src/internal/utils/bucketHelper.ts

```diff
@@ -399,6 +399,18 @@ export function getMeasuresFromMdObject(mdObject: VisualizationObject.IVisualiza
     }, []);
 }
 
+export function getBucketItemsByTypeFromMdObject(
+    mdObject: VisualizationObject.IVisualizationObjectContent,
+    type: string,
```

Comment:
type is here used as a bucket.localIdentifier, eg. "view", should not the input param and whole function have different name?

Reply:
Fixed by removing it.